libclamav/dsig.c

```diff
@@ -81,37 +81,59 @@ static char cli_ndecode(unsigned char value)
     return -1;
 }
 
-static unsigned char *cli_decodesig(const char *sig, unsigned int plen, fp_int e, fp_int n)
+static unsigned char *cli_decodesig(const char *sig, unsigned int plen, BIGNUM *e, BIGNUM *n)
 {
     int i, slen = strlen(sig), dec;
     unsigned char *plain;
-    fp_int r, p, c;
+    BIGNUM *r, *p, *c;
```

Comment:
Please initialize these to NULL.

Reply:
okay, but they are initialized a line later.